### PR TITLE
Add javascript support

### PR DESCRIPTION
**Pull request by MelaineGerard**
With this PR, I've added the support for Javascript for this course

### CODE REVIEW

File: starter_templates/javascript/code/src/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const filename = programArgs[1];
+
+if (command !== "tokenize") {
+    console.error(`Usage: Unknown command: ${command}`);
+    process.exit(1);
+}
+
+const fileContent = fs.readFileSync(filename, "utf8");
+
+// if (fileContent && fileContent.length !== 0) {
```
Comment (pawk3k) [inline starter_templates/javascript/code/src/index.js:23]: Sorry for nit picking but is this commented code required

**Comment by MelaineGerard** [comment]
With my last commit, I fixed all the errors in the CI

**Comment by MelaineGerard** [comment]
The PR is ready :)

**Comment by rohitpaulk** [comment]
Sorry for the wait here! We've been working on revamping language support to be more standardized across challenges (and to require less manual work in keeping this in sync). I'm just waiting on a few of those changes to go out before we take a look at this 🙂

**Comment by rohitpaulk** [comment]
Thanks @MelaineGerard!

**Pull request merged**